test/error_test.dart

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2015, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+library analyzer_cli.test.error;
+
+import 'package:unittest/unittest.dart';
+
+import 'utils.dart';
+
+void main() {
+  test("a valid Dart file doesn't throw any errors", () {
+    expect(errorsForFile('void main() => print("Hello, world!");'), isNull);
+  });
+
+  test("an empty Dart file doesn't throw any errors", () {
+    expect(errorsForFile(''), isNull);
+  });
+
+  test("an error on the first line", () {
+    expect(errorsForFile('void foo;\n'),
+        equals("Error in test.dart: Variables cannot have a type of 'void'\n"));
```

Comment:
Not sure.  It's a good point.  My leaning is to land these (as a literal port) and then revisit all the tests as we add functionality.  That said, if you do find the right answer in the meantime, I'm all ears! ;)